### DISCUSSION TITLE
June 2022 - First stable release

### DISCUSSION THREAD

**Discussion by Rafiot** [pandora-analysis/pandora#78] [Announcements]
This is the first official stable open-source release of Pandora. It has been tested relatively extensively over the last few months, but there might still be issues. If anything goes wrong, please open an issue and we will do our best to solve it.

# Core functionalities

* Easy to implement workers to analyze specific file formats, or connect to third party services
* Admin interface
* Session-based user management interface, and sharing
* Generate a preview of the submitted document (if applicable)
* Extract indicators/observables from submitted files
* Extract content of archives 
* Extract attachments from email in EML and MSG format
* Extract text content (if applicable)
* Extract EXIF metadata 
* Pool service to fetch emails from an IMAP mailbox
* Notify Administrator
* MISP export and submission (admin only)
* Statistics (admin only)
* Role management (admin only)
* Locally defined observables (legitimate/suspicious) (admin only)

<hr /><em>This discussion was created from the release <a href='https://github.com/pandora-analysis/pandora/releases/tag/v1.0.0'>June 2022 - First stable release</a>.</em>

**Comment by mmmaaaxxx** [comment]
Hello,

Do you plan to provide a docker image for pandora?

Regards

**Reply by Rafiot** [reply]
Hi, have you tried this one: https://github.com/pandora-analysis/pandora/pkgs/container/pandora

**Reply by mmmaaaxxx** [reply]
Thanks it's works great!!!
